src/test/v1.2/basic/fdc3.joinChannel.ts

```diff
@@ -43,5 +42,21 @@ export default () =>
       } else {
         assert.fail("No system channels available");
       }
+
+      await fdc3.joinChannel(channels[0].id);
+
+      const current = await fdc3.getCurrentChannel();
+
+      expect(current).to.not.be.null;
```

Comment:
duplicated code is back again
```suggestion
```

Reply:
This branch was created on top of Rob's PR. I'll update it since it's accepted and merged